src/main/resources/hapi.properties

```diff
@@ -11,7 +11,7 @@ fhir_version=R4
 # (the web UI similar to the one at http://hapi.fhir.org) will use to
 # connect internally to the FHIR server, so this also needs to be a name
 # accessible from the server itself.
-server_address=http://localhost:8080/hapi-fhir-jpaserver/fhir/
+server_address=http://localhost:8080/fhir
```

Comment:
Could we move this property to the yaml file and then just delete this file? Seems confusing to have config split across the two

Reply:
hapi.properties will be deleted as soon as I have ported the tests

Reply:
Hi @jamesagnew - I should be done now - however, there's still an issue in the tests. Would you mind having a look at whether I've wired everything up correctly in regards to the web socket subscriptions. It seems like the server does not emit events.